sklearn/decomposition/_pca.py

```diff
@@ -456,7 +464,11 @@ def _fit(self, X):
         if self._fit_svd_solver == "full":
             return self._fit_full(X, n_components)
         elif self._fit_svd_solver in ["arpack", "randomized"]:
-            return self._fit_truncated(X, n_components, self._fit_svd_solver)
+            return self._fit_truncated(
+                X,
+                n_components,
+                self._fit_svd_solver,
+            )
```

Comment:
I don't understand why black is not complaining about that...

Please keep the original formatting

Reply:
I have used black to compiled the code, the output is still the style. 
I am not sure why compilation will not be passed if I change it.

Reply:
I also think this patch must be reverted if it does not change the semantic, which is the case here.

Reply:
This hunk must be reverted.